content/en/docs/reference/config/analysis/ist0103/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: PodMissingProxy
+layout: analysis-message
+---
+
+This message occurs when the sidecar is not present or does not contain a configuration element that Istio expects.
```

Comment:
>  does not contain a configuration element that Istio expects

Is this how the analyzer actually behaves?

Reply:
fixed